src/main/menu/SkinSelectionPanel.java

```diff
@@ -144,14 +146,17 @@ private void getSelectButton(int i) {
 
 	private void getImageSkin(int i, int j) {
 
+		GameSettings settings;

```

Comment:
GameSettings is supposed to be an argument of the SkinSelectionPanel constructor.